e2e_tests/tests/fixtures/tf_keras_grad_aggregation/grad_aggregation_test.py

```diff
@@ -11,16 +13,26 @@
     pass
 
 
+_PRE_TF_2_4_0 = LooseVersion(tf.__version__) < LooseVersion("2.4.0")
+
+
 class CustomOptimizer(optimizer_v2.OptimizerV2):
     def get_config(self) -> Any:
         config = super(CustomOptimizer, self).get_config()
         return config
 
+    def _create_slots(self, var_list: Any) -> None:
+        pass
+
+    def _resource_apply_dense(self, grad: Any, var: Any, apply_state: Any = None) -> Any:
```

Comment:
Can we use a more descriptive type annotation here?

Reply:
fairly sure mypy interprets `tf.Tensors` and `tf.Variable` as `Any`